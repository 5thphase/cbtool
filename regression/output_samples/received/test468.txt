 status: VMC 9F3AAFB2-9D87-5F21-842D-32BFE6EA0A71 was successfully unregistered on SimCloud "TESTCLOUD".
VMC object 9F3AAFB2-9D87-5F21-842D-32BFE6EA0A71 (named "simzone_a") was sucessfully detached from this experiment.
The following ATTACHED VMCs are in this experiment (Cloud TESTCLOUD) :
|name                    |host_count      |pool                    |cloud_hostname                  |cloud_ip         
|simzone_b               |5               |sut                     |simzone_b                       |225.214.181.32   
|simzone_c               |5               |sut                     |simzone_c                       |99.151.49.188    
|simzone_d               |5               |lg                      |simzone_d                       |151.59.51.188    

The following statistics are available on this experiment (Cloud TESTCLOUD) :
------------------------- OBJECT STORE -----------------------
|Metric                                               |Value                         
|Used Memory                                          |1.53M                         
|Redis Version                                        |2.6.16                        
|Uptime (in seconds)                                  |10441564                      
|Total Connections Received                           |230911                        
|Total Commands Processed                             |26356987                      
|Number of Keys                                       |231                           
------------------------- METRIC STORE -----------------------
|Metric                                               |Value                         
|MongoDB Version                                      |2.4.9                         
|Storage Size                                         |3936256                       
|Data Size                                            |445432                        
|Index Size                                           |367920                        
|Average Object Size                                  |510.231386025                 
|Collections                                          |13                            
--------------------- EXPERIMENT OBJECTS ---------------------
|Object                                               |Count                         
|VMCs                                                 |3                             
|HOSTs                                                |15                            
|VMs                                                  |0                             
|AIs                                                  |0                             
|AIDRSs                                               |0                             
|VMCRSs                                               |0                             
|FIRSs                                                |0                             
------------------ EXPERIMENT-WIDE COUNTERS ------------------
|Counter                                              |Value                         
|VMC RESERVATIONS                                     |3                             
|VMCs ARRIVED                                         |8                             
|VMCs ARRIVING                                        |0                             
|VMCs DEPARTED                                        |5                             
|VMCs DEPARTING                                       |0                             
|VMCs FAILED                                          |0                             
|HOST RESERVATIONS                                    |0                             
|HOSTs ARRIVED                                        |0                             
|HOSTs ARRIVING                                       |0                             
|HOSTs DEPARTED                                       |0                             
|HOSTs DEPARTING                                      |0                             
|HOSTs FAILED                                         |0                             
|VM RESERVATIONS                                      |0                             
|VMs ARRIVED                                          |0                             
|VMs ARRIVING                                         |0                             
|VMs CAPTURING                                        |0                             
|VMs DEPARTED                                         |0                             
|VMs DEPARTING                                        |0                             
|VMs FAILED                                           |0                             
|AI RESERVATIONS                                      |0                             
|AIs ARRIVED                                          |0                             
|AIs ARRIVING                                         |0                             
|AIs CAPTURING                                        |0                             
|AIs DEPARTED                                         |0                             
|AIs DEPARTING                                        |0                             
|AIs FAILED                                           |0                             
|AIDRS RESERVATIONS                                   |0                             
|AIDRSs ARRIVED                                       |0                             
|AIDRSs ARRIVING                                      |0                             
|AIDRSs DEPARTED                                      |0                             
|AIDRSs DEPARTING                                     |0                             
|AIDRSs FAILED                                        |0                             
|VMCRS RESERVATIONS                                   |0                             
|VMCRSs ARRIVED                                       |0                             
|VMCRSs ARRIVING                                      |0                             
|VMCRSs DEPARTED                                      |0                             
|VMCRSs DEPARTING                                     |0                             
|VMCRSs FAILED                                        |0                             
|FIRS RESERVATIONS                                    |0                             
|FIRSs ARRIVED                                        |0                             
|FIRSs ARRIVING                                       |0                             
|FIRSs DEPARTED                                       |0                             
|FIRSs DEPARTING                                      |0                             
|FIRSs FAILED                                         |0                             
|EXPERIMENT COUNTER                                   |38                            

 ---------------- VMC-WIDE COUNTERS ----------------
|9987B189-7E0A-5940-A2F9-46FF276E228B (simzone_b) VM RESERVATIONS|0                             
|2B964689-8BEC-52E2-97C4-BA622EB43467 (simzone_d) VM RESERVATIONS|0                             
|FECFFB9A-ADAE-57F2-8DCD-FBD0004CB05E (simzone_c) VM RESERVATIONS|0                             

