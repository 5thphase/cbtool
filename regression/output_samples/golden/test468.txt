 status: VMC 35444C87-852A-5B12-B5D5-466EE2A8AF58 was successfully unregistered on SimCloud "TESTCLOUD".
VMC object 35444C87-852A-5B12-B5D5-466EE2A8AF58 (named "simzone_a") was sucessfully detached from this experiment.
The following ATTACHED VMCs are in this experiment (Cloud TESTCLOUD) :
|name                    |host_count      |pool                    |cloud_hostname                  |cloud_ip         
|simzone_c               |5               |sut                     |simzone_c                       |201.75.193.74    
|simzone_b               |5               |sut                     |simzone_b                       |244.137.114.126  
|simzone_d               |5               |lg                      |simzone_d                       |227.230.56.204   

The following statistics are available on this experiment (Cloud TESTCLOUD) :
------------------------- OBJECT STORE -----------------------
|Metric                                               |Value                         
|Used Memory                                          |1.99M                         
|Redis Version                                        |2.5.12                        
|Uptime (in seconds)                                  |2417260                       
|Total Connections Received                           |4537                          
|Total Commands Processed                             |3873834                       
|Number of Keys                                       |231                           
------------------------- METRIC STORE -----------------------
|Metric                                               |Value                         
|MongoDB Version                                      |2.4.5                         
|Storage Size                                         |5448130560.0                  
|Data Size                                            |4961866848.0                  
|Index Size                                           |133669424                     
|Average Object Size                                  |8105.08279755                 
|Collections                                          |39                            
--------------------- EXPERIMENT OBJECTS ---------------------
|Object                                               |Count                         
|VMCs                                                 |3                             
|HOSTs                                                |15                            
|VMs                                                  |0                             
|AIs                                                  |0                             
|AIDRSs                                               |0                             
|VMCRSs                                               |0                             
|FIRSs                                                |0                             
------------------ EXPERIMENT-WIDE COUNTERS ------------------
|Counter                                              |Value                         
|VMC RESERVATIONS                                     |3                             
|VMCs ARRIVED                                         |8                             
|VMCs ARRIVING                                        |0                             
|VMCs DEPARTED                                        |5                             
|VMCs DEPARTING                                       |0                             
|VMCs FAILED                                          |0                             
|HOST RESERVATIONS                                    |0                             
|HOSTs ARRIVED                                        |0                             
|HOSTs ARRIVING                                       |0                             
|HOSTs DEPARTED                                       |0                             
|HOSTs DEPARTING                                      |0                             
|HOSTs FAILED                                         |0                             
|VM RESERVATIONS                                      |0                             
|VMs ARRIVED                                          |0                             
|VMs ARRIVING                                         |0                             
|VMs CAPTURING                                        |0                             
|VMs DEPARTED                                         |0                             
|VMs DEPARTING                                        |0                             
|VMs FAILED                                           |0                             
|AI RESERVATIONS                                      |0                             
|AIs ARRIVED                                          |0                             
|AIs ARRIVING                                         |0                             
|AIs CAPTURING                                        |0                             
|AIs DEPARTED                                         |0                             
|AIs DEPARTING                                        |0                             
|AIs FAILED                                           |0                             
|AIDRS RESERVATIONS                                   |0                             
|AIDRSs ARRIVED                                       |0                             
|AIDRSs ARRIVING                                      |0                             
|AIDRSs DEPARTED                                      |0                             
|AIDRSs DEPARTING                                     |0                             
|AIDRSs FAILED                                        |0                             
|VMCRS RESERVATIONS                                   |0                             
|VMCRSs ARRIVED                                       |0                             
|VMCRSs ARRIVING                                      |0                             
|VMCRSs DEPARTED                                      |0                             
|VMCRSs DEPARTING                                     |0                             
|VMCRSs FAILED                                        |0                             
|FIRS RESERVATIONS                                    |0                             
|FIRSs ARRIVED                                        |0                             
|FIRSs ARRIVING                                       |0                             
|FIRSs DEPARTED                                       |0                             
|FIRSs DEPARTING                                      |0                             
|FIRSs FAILED                                         |0                             
|EXPERIMENT COUNTER                                   |38                            

 ---------------- VMC-WIDE COUNTERS ----------------
|80FAE389-CBF7-5975-94FD-8D5CD85D723E (simzone_b) VM RESERVATIONS|0                             
|BDEF1BDD-FF65-58F0-BAC9-26D21728F092 (simzone_d) VM RESERVATIONS|0                             
|2905F1A8-EC2A-5B51-A60B-F22EA13FA5E3 (simzone_c) VM RESERVATIONS|0                             

